js-apps/meep-frontend/src/js/containers/exec/dashboard-container.js

```diff
@@ -0,0 +1,783 @@
+import { connect } from 'react-redux';
```

Comment:
Missing header. Make sure the Apache 2.0 header + IDCC copyright is on all files.

Reply:
Added missing headers.